static_src/components/app_container.jsx

```diff
@@ -60,12 +61,14 @@ export default class AppContainer extends React.Component {
     AppStore.addChangeListener(this._onChange);
     OrgStore.addChangeListener(this._onChange);
     SpaceStore.addChangeListener(this._onChange);
+    pageMetadataActions.loadPage('app');
```

Comment:
Yea you would essentially have to operate similar to the side nav, unload the page on every route that doesn't have a title?

Also something to keep in mind: are there any pages left without a title now that marketplace and the space users, services routes are gone?